ocs_ci/utility/framework/base_initializer.py

```
@@ -0,0 +1,201 @@
+        framework.config.init_cluster_configs()
+        load_config(args.conf)
+
+        # Updating resource_checker to False since it's not needed
+        config.RUN["resource_checker"] = False
+
+        logger.debug("Verifying cluster_name and cluster_path")
+        cluster_name = args.cluster_name
+        cluster_path = os.path.expanduser(args.cluster_path)
+
+        if not os.path.exists(cluster_path):
+            raise FileNotFoundError(f"No such directory: {cluster_path}")
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)